src/result.rs

```diff
@@ -64,8 +64,22 @@ pub enum WebSocketOtherError {
 
 impl fmt::Display for WebSocketOtherError {
 	fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
-		fmt.write_str("WebSocketError: ")?;
-		fmt.write_str(self.description())?;
+		match self {
+			WebSocketOtherError::RequestError(e) => write!(fmt, "WebSocket request error: {}", e)?,
+			WebSocketOtherError::ResponseError(e) => write!(fmt, "WebSocket request error: {}", e)?,
```

Comment:
Maybe use `response` here?
```suggestion
			WebSocketOtherError::ResponseError(e) => write!(fmt, "WebSocket response error: {}", e)?,
```

Reply:
Good catch, thanks!